### DISCUSSION TITLE
Weird version.txt not found error on github actions

### DISCUSSION THREAD

**Discussion by aliencaocao** [pyinstaller/pyinstaller#7531] [Help] [answered]
ls shows file is indeed there: https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/actions/runs/4575863831/jobs/8079251361#step:6:15
But pyinstaller somehow complains it's not there:
https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/actions/runs/4575863831/jobs/8079251361#step:7:296

I am very confused right now as I cannot reproduce the issue locally.

**Comment by rokm** [comment, marked as answer]
Seems to me that your "PyInstaller Windows" step cleans the repository and removes the version.txt file:
https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/actions/runs/4575863831/jobs/8079251361#step:7:76

```
Cleaning the repository
  "C:\Program Files\Git\bin\git.exe" clean -ffdx
  Removing src/version.txt
  "C:\Program Files\Git\bin\git.exe" reset --hard HEAD
  HEAD is now at 93be09b Attempt to fix windows build
```

**Reply by aliencaocao** [reply]
Thanks!
